app/src/main/java/zapsolutions/zap/util/PrefsUtil.java

```diff
@@ -26,6 +26,9 @@
     public static final String PREFS_ENCRYPTED = "prefs_encrypted";
     public static final String WALLET_CONFIGS = "wallet_configs";
 
+    // default values
+    public static final String DEFAULT_FIAT_UNITS = "[]";
```

Comment:
Sorry I wasn't specific enough, I meant to make a constant for the `fiat_available` String.

If we replace
`PrefsUtil.getPrefs().getString("fiat_available",PrefsUtil.DEFAULT_FIAT_UNITS).equals(PrefsUtil.DEFAULT_FIAT_UNITS)`
with
`PrefsUtil.getPrefs().contains("fiat_available")`
then we are able to remove the `"[]"`.

Reply:
Actually it was specific, I just didn't read carefully...
I now changed that.